libs/auth-api-lib/src/lib/resources/models/api-scope-group.model.ts

```diff
@@ -67,16 +64,6 @@ export class ApiScopeGroup extends Model {
   })
   order!: number
 
-  @Column({
-    type: DataType.STRING,
-    allowNull: false,
-  })
-  @ApiProperty({
-    example: '@island.is',
-  })
-  @ForeignKey(() => Domain)
-  domainName!: string
```

Comment:
The api scope group needs also to belong to just a single domain.

Reply:
Fixed.